process/customer-success.md

```diff
@@ -0,0 +1,409 @@
+[[TOC]]
+
+# Introduction
+
+This document describes MayaData’s Customer Success Advocacy Team/Organization structure and processes to achieve customer success as we expand product offerings and grow our customers. 
+
+For the first 5 customers, current team members of MayaData will fill in the roles/processes described in this document. As we expand to more customers in the later half of 2019-Q2, we will likely create a dedicated Customer Success Team and enhanced processes as described in this document. 
+
+The content in this document is derived from past experiences as well as learning from the following online resources:
+
+* [https://www.helpscout.com/helpu/customer-support-department/](https://www.helpscout.com/helpu/customer-support-department/)
+* [https://www.zendesk.com/resources/structure-customer-support-organization/](https://www.zendesk.com/resources/structure-customer-support-organization/)
+* [https://www.mmcventures.com/wp-content/uploads/2018/05/MMC-Ventures-Customer-Service-Playbook-single-pages.pdf](https://www.mmcventures.com/wp-content/uploads/2018/05/MMC-Ventures-Customer-Service-Playbook-single-pages.pdf)
+
+# Customer Success with MayaData
+
+The PLOW culture at MayaData emphasizes that entire company MUST empathise with the Customer and make the Customer win within his/her Organization.  
+
+The single most important objective that the whole company strives for is to **_Deliver Data Agility - Free DevOps and Kubernetes from storage constraints using OpenEBS, validate with Litmus by chaos engineering and monitor with MayaOnline_***.*
+
+Our design philosophy is to deliver radically simple solutions that are open source based that enable developers and DevOps end user self service success.  We then sell solutions that wrap around these solutions to enable enterprises to control, manage, and scale their operations.  
+
+MayaData has chosen Chat as the primary mode of customer support, since Chat implicitly sets the expectations for a shorter response time compared to emails and also maintains a record of the interactions as opposed to Phone conversations. MayaData sets the bar for quick response very high compared to other companies - at less than 15 minutes.   Our goal is to be a trusted partner to our users - **_whether they are paid customer or community user_** - as they seek to achieve data agility.  Through this trusted relationship we can out learn alternatives while also building customer trust and loyalty.
```

Comment:
In addition to chat, especially for enterprise edition support, we will also use the popular ticket management and knowledge base tools such as ZenDesk.